sphinx/screens-detail/tribe-members-list/tribe-members-list/src/main/java/chat/sphinx/tribe_members_list/ui/viewstate/TribeMemberHolderViewState.kt

```diff
@@ -0,0 +1,29 @@
+package chat.sphinx.tribe_members_list.ui.viewstate
+
+import chat.sphinx.concept_network_query_contact.model.ContactDto
+
+internal sealed class TribeMemberHolderViewState(
+    val contactDto: ContactDto? = null,
+    val showInitial: Boolean = false
```

Comment:
arguments would be much better served as a subssealed class to hold the Pending/Memeber subtypes.

We should also be utilizing the `ViewState` to describe the actual view's state instead of passing a `ContactDto` and then analyzing it in the Fragment. That analysis can either happen here in the class via utilizing an operator function to set the necessary values for the respective views, or in the ViewModel for where it gets set.